lib/ansible/cli/galaxy.py

```diff
@@ -22,8 +22,14 @@
 from ansible.errors import AnsibleError, AnsibleOptionsError
 from ansible.galaxy import Galaxy, get_collections_galaxy_meta_info
 from ansible.galaxy.api import GalaxyAPI
-from ansible.galaxy.collection import build_collection, install_collections, publish_collection, \
-    validate_collection_name
+from ansible.galaxy.collection import (
+    _find_existing_collections,
```

Comment:
Maybe this should be renamed to remove the `_`

Reply:
Done in #66577